api-objects/Appointment.ts

```diff
@@ -34,4 +37,28 @@ export class Appointment {
     public ths_sectionc: number;
     public ths_sectionc_example: string;
     public ts_type: string;
+
+    public createTfiMap() {
```

Comment:
These would be better as const static fields.

Reply:
What would the correct syntax for adding these as const static fields?